### DISCUSSION TITLE
Generate and publish rss feeds

### DISCUSSION THREAD

**Discussion by markisevil** [manoelcampos/asciidoctor-ghpages-action#39] [Ideas]
Could we add rss feed generation - maybe something like https://github.com/jekyll/jekyll-feed ?

**Comment by manoelcampos** [comment]
That is for a Jekyll website (which have specific actions for building such sites).
My action is not intended to be used in such a structured websites like that.
You can define the site structure as you which and configure the action to build it.
It's intended for more basic sites.
I don't see how we could generate a RSS feed if the sites built with my action
don't have a specific structure, such as a post list to create a feed from.

**Reply by markisevil** [reply]
I see what you mean, and it's absolutely fair to say it's not in scope here. 

I tried searching for an Atom or RSS feed converter for Asciidoctor as well, and didn't find one, so it's perhaps a more general Asciidoctor question (that could later be applied here too).

**Reply by markisevil** [reply]
There's a semantic part to it too: should a static website generator provide a dynamic feed?

**Reply by manoelcampos** [reply]
If you have a Jekyll site, despite it being static, when you push changes to GitHub, a GitHub Action can update the RSS feed automatically.

**Reply by manoelcampos** [reply]
Probably you just have to enable the jekyll-feed plugin you mentioned.
